internal/experimental/registry/client.go

```diff
@@ -495,6 +518,11 @@ func (c *Client) Push(data []byte, ref string, options ...PushOption) (*PushResu
 	}
 	fmt.Fprintf(c.out, "Pushed: %s\n", result.Ref)
 	fmt.Fprintf(c.out, "Digest: %s\n", result.Manifest.Digest)
+	if strings.Contains(result.Ref, "_") {
```

Comment:
The `result.Ref` is the full URI to the chart in a registry. The `_` character is a valid path character. This will detect cases outside of the tag. Shouldn't it be scoped to the tag?

Reply:
good catch, updated in fed0d31